api/app/src/TestHelpers/CourtOrderTestHelper.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
We've got a createCourtOrder method on the Fixtures class which could be used for this.

There's also a method on the Deputy class to associate a deputy with a court order (associateWithCourtOrder()), which would replace the second method.